docs/faq.md

```diff
@@ -338,4 +338,10 @@ import plotly.graph_objects as go
 fig = go.Figure(layout=dict(template='plotly'))
 ```
 
+</details>
+<details>
+<summary><h3 style={{margin: '0px 0px 0px 0px'}}>My `Card` is not fitting its content, what do I do?</h3></summary>
```

Comment:
can you maybe add an image as an example?

Reply:
I would also suggest adding the code fix example

Reply:
Good points, have added both